### PR TITLE
Feature/issue 65/admin page

### PR DESCRIPTION
**Pull request by BayanAlsalem**

### CODE REVIEW

**Review by Firdaws-b — APPROVED** [review]

**Review by fluffyCookie0 — APPROVED** [review]

File: web-client/src/pages/AdminProfile.js
Comment (ItsDezi) [inline web-client/src/pages/AdminProfile.js:1]: Please fix routing to this page before merging

File: web-client/src/App.js
Comment (ItsDezi) [inline web-client/src/App.js:1]: Please ensure you lint big files that many components rely on like this one.

File: web-client/src/Components/NavBars/AdminSideBar.js
Comment (ItsDezi) [inline web-client/src/Components/NavBars/AdminSideBar.js:1]: Can you not reuse the existing sidebar class and just add another export?

File: web-client/src/pages/Home.js
Comment (ItsDezi) [inline web-client/src/pages/Home.js:1]: after logging in as an admin, nothing appears on home page for me

**Review by ItsDezi — CHANGES_REQUESTED** [review]

File: web-client/src/pages/AdminProfile.js
Comment (BayanAlsalem) [reply web-client/src/pages/AdminProfile.js:1]: issue fixed. Please refer to commit number c8d45b3

File: web-client/src/pages/Home.js
Comment (BayanAlsalem) [reply web-client/src/pages/Home.js:1]: issue fixed. Please refer to commit number c8d45b3

File: web-client/src/App.js
Comment (BayanAlsalem) [reply web-client/src/App.js:1]: issue fixed. Please refer to commit number c8d45b3

File: web-client/src/Components/NavBars/AdminSideBar.js
Comment (BayanAlsalem) [reply web-client/src/Components/NavBars/AdminSideBar.js:1]: I'll look at this in sprint 4

File: web-client/src/pages/Home.js
Comment (ItsDezi) [reply web-client/src/pages/Home.js:1]: why was this happening? @BayanAlsalem

File: web-client/src/pages/jobPost.js
Comment (ItsDezi) [inline web-client/src/pages/jobPost.js:1]: Should we leave these edits out. I made a lot of modifications to this page today regarding the editing feature and I feel like admins should also be able to edit/delete postings.

**Review by ItsDezi — CHANGES_REQUESTED** [review]

**Review by ItsDezi — APPROVED** [review]

**Pull request merged**